package.json

```diff
@@ -20,7 +20,9 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "lint": "yarn eslint --quiet \"src/**/*.{ts,tsx}\"",
+    "lint:format": "yarn lint --fix"
   },
   "dependencies": {
```

Comment:
各種パッケージのインストール
```bash
$ yarn add -D @babel/core @babel/eslint-parser eslint-plugin-import
```

ログ
```
[1/4] Resolving packages...
[2/4] Fetching packages...
warning Pattern ["eslint-plugin-import@^2.29.1"] is trying to unpack in the same destination "/home/node/.cache/yarn/v6/npm-eslint-plugin-import-2.29.1-d45b37b5ef5901d639c15270d74d46d161150643-integrity/node_modules/eslint-plugin-import" as pattern ["eslint-plugin-import@^2.26.0"]. This could result in non-deterministic behavior, skipping.
[3/4] Linking dependencies...
warning " > eslint-config-weseek@2.1.1" has incorrect peer dependency "@typescript-eslint/eslint-plugin@^4.28.5".
warning " > eslint-config-weseek@2.1.1" has incorrect peer dependency "@typescript-eslint/parser@^4.28.5".
warning " > eslint-config-weseek@2.1.1" has incorrect peer dependency "eslint@^7.32.0".
warning " > eslint-config-weseek@2.1.1" has incorrect peer dependency "eslint-import-resolver-typescript@^2.4.0".
warning " > eslint-config-weseek@2.1.1" has unmet peer dependency "eslint-plugin-vue@^7.15.0".
warning "eslint-config-weseek > eslint-config-airbnb@17.1.1" has incorrect peer dependency "eslint@^4.19.1 || ^5.3.0".
warning "eslint-config-weseek > eslint-config-airbnb@17.1.1" has unmet peer dependency "eslint-plugin-jsx-a11y@^6.2.3".
warning "eslint-config-weseek > eslint-config-airbnb > eslint-config-airbnb-base@13.2.0" has incorrect peer dependency "eslint@^4.19.1 || ^5.3.0".
[4/4] Building fresh packages...
success Saved lockfile.
success Saved 5 new dependencies.
info Direct dependencies
├─ @babel/core@7.23.7
├─ @babel/eslint-parser@7.23.3
└─ eslint-plugin-import@2.29.1
info All dependencies
├─ @babel/core@7.23.7
├─ @babel/eslint-parser@7.23.3
├─ @babel/helpers@7.23.8
├─ @nicolo-ribaudo/eslint-scope-5-internals@5.1.1-v1
└─ eslint-plugin-import@2.29.1
```

このうち `unmet peer dependency` の警告が出たパッケージ
- eslint-plugin-vue@^7.15.0"
- eslint-plugin-jsx-a11y@^6.2.3"

上記パッケージを含めて再度インストール
```bash
yarn add -D eslint-plugin-vue@^7.15.0 eslint-plugin-jsx-a11y@^6.2.3 @babel/core @babel/eslint-parser eslint-plugin-import
```

ログ
```
yarn add v1.22.19
[1/4] Resolving packages...
[2/4] Fetching packages...
warning Pattern ["eslint-plugin-import@^2.29.1"] is trying to unpack in the same destination "/home/node/.cache/yarn/v6/npm-eslint-plugin-import-2.29.1-d45b37b5ef5901d639c15270d74d46d161150643-integrity/node_modules/eslint-plugin-import" as pattern ["eslint-plugin-import@^2.26.0"]. This could result in non-deterministic behavior, skipping.
[3/4] Linking dependencies...
warning " > eslint-config-weseek@2.1.1" has incorrect peer dependency "@typescript-eslint/eslint-plugin@^4.28.5".
warning " > eslint-config-weseek@2.1.1" has incorrect peer dependency "@typescript-eslint/parser@^4.28.5".
warning " > eslint-config-weseek@2.1.1" has incorrect peer dependency "eslint@^7.32.0".
warning " > eslint-config-weseek@2.1.1" has incorrect peer dependency "eslint-import-resolver-typescript@^2.4.0".
warning "eslint-config-weseek > eslint-config-airbnb@17.1.1" has incorrect peer dependency "eslint@^4.19.1 || ^5.3.0".
warning "eslint-config-weseek > eslint-config-airbnb > eslint-config-airbnb-base@13.2.0" has incorrect peer dependency "eslint@^4.19.1 || ^5.3.0".
[4/4] Building fresh packages...
success Saved lockfile.
success Saved 9 new dependencies.
info Direct dependencies
├─ @babel/core@7.23.7
├─ @babel/eslint-parser@7.23.3
├─ eslint-plugin-import@2.29.1
└─ eslint-plugin-vue@7.20.0
info All dependencies
├─ @babel/core@7.23.7
├─ @babel/eslint-parser@7.23.3
├─ @babel/helpers@7.23.8
├─ @nicolo-ribaudo/eslint-scope-5-internals@5.1.1-v1
├─ eslint-plugin-import@2.29.1
├─ eslint-plugin-vue@7.20.0
├─ eslint-utils@2.1.0
├─ lodash@4.17.21
└─ vue-eslint-parser@7.11.0
Done in 2.90s.
```